tools/bazelify_tests/test/BUILD

```diff
@@ -59,29 +59,28 @@ grpc_run_tests_py_test(
     docker_image_version = "tools/dockerfile/test/cxx_debian11_x64.current_version",
 )
 
-# TODO(jtattermusch): Reintroduce ruby tests once they pass.
-# # Ruby
-# grpc_run_tests_py_test(
-#     name = "runtests_ruby_linux_dbg",
-#     size = "enormous",
-#     args = [
-#         "-l ruby -c dbg",
-#     ],
-#     docker_image_version = "tools/dockerfile/test/ruby_debian11_x64.current_version",
-#     prepare_script = ":prepare_ruby.sh",
-#     use_login_shell = True,  # ruby's docker image uses RVM which wierdly requires login shell
-# )
-
-# grpc_run_tests_py_test(
-#     name = "runtests_ruby_linux_opt",
-#     size = "enormous",
-#     args = [
-#         "-l ruby -c opt",
-#     ],
-#     docker_image_version = "tools/dockerfile/test/ruby_debian11_x64.current_version",
-#     prepare_script = ":prepare_ruby.sh",
-#     use_login_shell = True,  # ruby's docker image uses RVM which wierdly requires login shell
-# )
+# Ruby
+grpc_run_tests_py_test(
+    name = "runtests_ruby_linux_dbg",
+    size = "enormous",
+    args = [
+        "-l ruby -c dbg",
+    ],
+    docker_image_version = "tools/dockerfile/test/ruby_debian11_x64.current_version",
+    prepare_script = ":prepare_ruby.sh",
+    use_login_shell = True,  # ruby's docker image uses RVM which wierdly requires login shell
+)
+
+grpc_run_tests_py_test(
```

Comment:
i know this isn't specific to this PR (maybe an an out-of-band clean-up), but I'm realizing that the suffix "_py_test" implies that this is a test for python code. Compare `py_test`, `py_strict_test`, etc. Maybe we could find a different name. Regardless, not super high priority and does not block merge of this PR.

Reply:
Good catch, I'll try to come up with something better as a followup.